src/axolotl/utils/data.py

```diff
@@ -336,6 +352,18 @@ def for_d_in_datasets(dataset_configs):
                         split=None,
                         storage_options=storage_options,
                     )
+            elif ds_from_puree:
+                ds_type = get_ds_type(config_dataset)
+                dataset_id = config_dataset.path.split("://")[1]
```

Comment:
wouldn't `config_dataset.path.split("://")[1]` be `datasets/DATASET_ID`??

Reply:
the path for puree is supposed to be `puree://<dataset-id>` will add it on the docs